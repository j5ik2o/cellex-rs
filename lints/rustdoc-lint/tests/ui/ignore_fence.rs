#![allow(dead_code)]

/// ```ignore
/// let value = 1;
/// ```
pub fn sample() {}

fn main() {}
