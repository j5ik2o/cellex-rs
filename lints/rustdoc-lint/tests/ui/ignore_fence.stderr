warning: rustdoc のコードフェンスでは `ignore` を使用できません
  --> $DIR/ignore_fence.rs:3:5
   |
LL | /// ```ignore
   |     ^^^^^^^^^
   |
   = help: サンプルコードは ` ```rust ` としてコンパイル可能な形で記述してください
   = note: AI向けアドバイス: 実行不能なコードは避け、最小限の `rust` サンプルを提示しましょう。
   = note: `#[warn(rustdoc_lint)]` on by default

warning: 1 warning emitted

